daprdocs/content/en/operations/performance-and-scalability/perf-service-invocation.md

```diff
@@ -0,0 +1,89 @@
+---
+type: docs
+title: "Service invocation performance"
+linkTitle: "Service invocation performance"
+weight: 10000
+description: ""
+---
+In this article you'll find an overview of the components needed to run Dapr in different hosting environments along with performance benchmarks and resource utilizations.
+
+## System overview
+
+Dapr consists of a data plane, the sidecar that runs next to your app, and a control plane that configures the sidecars and provides capabilities such as cert and identity management.
+
+### Self-hosted components
+
+* Sidecar (data plane)
+* Sentry (optional, control plane)
+* Placement (optional, control plane)
+
+For more information see [overview of Dapr in self-hosted mode]({{< ref self-hosted-overview.md >}}).
+
+### Kubernetes components
+
+* Sidecar (data plane)
+* Sentry (optional, control plane)
+* Placement (optional, control planee)
+* Operator (control plane)
+* Sidecar Injector (control plane)
+
+For more information see [overview of Dapr on Kubernetes]({{< ref kubernetes-overview.md >}}).
+
+## Performance summary for Dapr v0.11.3
+
+The service invocation API is a reverse proxy with built-in service discovery to connect to other services. This includes tracing, metrics, mTLS for in-transit encryption of traffic, together with resiliency in the form of retries for network partitions and connection errors.
+
+Using service invocation you can call from HTTP to HTTP, HTTP to gRPC, gRPC to HTTP, and gRPC to gRPC. Dapr does not use HTTP for the communication between sidecars, always using gRPC, while carrying over the semantics of the protocol used when called from the app. Service invocation is the underlying mechanism of communicating with Dapr Actors.
+
+For more information see [service invocation overview]({{< ref service-invocation-overview.md >}}).
+
+### Kubernetes performance test setup
+
+The test was conducted on a 3 node Kubernetes cluster, using commodity hardware running 4 cores and 8GB of RAM, without any network acceleration.
+The setup included a load tester ([Fortio](https://github.com/fortio/fortio)) pod with a Dapr sidecar injected into it that called the service invocation API to reach a pod on a different node.
+
+Test parameters:
+
+* 1000 requests per second
+* Sidecar limited to 0.5 vCPU
+* Sidecar mTLS enabled
+* Sidecar telemetry enabled (tracing with a sampling rate of 0.1)
+* Payload of 1KB
+
+The baseline test included direct, non-encrypted traffic, without telemetry, directly from the load tester to the target app.
+
+### Control plane performance
+
+The Dapr control plane uses a total of 0.01 vCPU and 77 Mb.
```

Comment:
How do you get these numbers from the table below? I add these up to 61.6 Mb. And should you not the total from vCPU be also be 0.01 (currently it adds to 0.009) vCPU

Reply:
Initially I counted the dashboard as part of the control plane.. updated the numbers.